lib/cli/cliPlugin.js

```diff
@@ -244,10 +244,10 @@ module.exports = class CLIPlugin extends CLICommand {
 
                     // Read the state
                     try {
-                        const enabled = await states.getState(pluginEnabledId)
-                            .val;
```

Comment:
This lead to cannout read property val of undefined thus default was applied everytime. But also with a promise, we cannot add property `val` of the promise result like done before